content/documentation/pages/2-concepts/0-index.md

```diff
@@ -6,3 +6,5 @@ summary: true
 ---
 
 # Concepts
+
+This section introduces the high level architecture of Data Flow as well as the architectural style of the Stream and Batch applications it creates and deploys.
```

Comment:
Can we use `Spring Cloud Data Flow` in all the places instead of just `Data Flow`?

Reply:
It gets a bit long/tiring to me to keep doing it over and over....